### PR TITLE
Add version param to get-daml.sh

### LINKED ISSUE #1461: Make get.daml.com take an optional version
Looking at #1459 you need slightly more knowledge to install a specific SDK version, which is unfortunate. I suggest we make it so you can do:

```
curl -ssL https://get.daml.com | sh -s 0.12.20
```

And then it really is easy to install specific versions.

### PR DESCRIPTION
**Pull request by associahedron**
Fixes #1461

### CODE REVIEW

File: daml-assistant/get-daml.sh [hunk 1]
```diff
@@ -10,7 +10,20 @@
 # For more information please visit https://daml.com/ and https://docs.daml.com/
 #
 
+#
+# USAGE:
+#    get-daml.sh            Download and install the latest DAML SDK release.
+#    get-daml.sh VERSION    Download and install given version of DAML SDK.
+#
+
+ARG1="$1"
```
Comment (cocreature) [inline daml-assistant/get-daml.sh]: I would just be explicit and use `ARG1="${1:-}" instead (or do that later when you reference it).

Comment (associahedron) [reply daml-assistant/get-daml.sh]: Thanks! :+1:

File: daml-assistant/get-daml.sh [hunk 2]
```diff
@@ -70,8 +81,8 @@ fi
 readonly TARBALL="daml-sdk-$VERSION-$OS.tar.gz"
 readonly URL="https://github.com/digital-asset/daml/releases/download/v$VERSION/$TARBALL"
 
-echo "$(tput setaf 3)Downloading latest DAML SDK. This may take a while.$(tput sgr 0)"
-curl -L $URL --output $TARBALL --progress-bar
+echo "$(tput setaf 3)Downloading DAML SDK. This may take a while.$(tput sgr 0)"
```
Comment (neil-da) [inline daml-assistant/get-daml.sh]: Could we say the version number on this line? That would be nice.

**Comment by neil-da** [comment]
@bolek-wisniewski - please switch to this version.

File: daml-assistant/get-daml.sh (hunk 2 above)
Comment (associahedron) [reply daml-assistant/get-daml.sh]: Sure! I'll make another PR since this one is already merged.

**Pull request merged**